api/handlers_test.go

```diff
@@ -236,23 +320,23 @@ func TestVerifySkappReport(t *testing.T) {
 
 // newTestAPI returns a new API instance
 func newTestAPI(dbName string, skyd skyd.API) (*API, error) {
-	// create a context with timeout
-	ctx, cancel := context.WithTimeout(context.Background(), database.MongoDefaultTimeout)
-	defer cancel()
-
 	// create a nil logger
 	logger := logrus.New()
 	logger.Out = ioutil.Discard
 
 	// create database
-	db, err := database.NewCustomDB(ctx, "mongodb://localhost:37017", dbName, options.Credential{
+	db, err := database.NewCustomDB(context.Background(), "mongodb://localhost:37017", dbName, options.Credential{
```

Comment:
Why is that?

Reply:
just a mistake that got committed once, we were handing out a database with a cancelled context, causing find operations to immediately time out.